gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -15,8 +15,10 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
     }
 
     private AndroidCommandPluginExtension createExtension() {
-        def project = ProjectBuilder.builder().build()
+	def projectDir = new File('..')
+        def project = ProjectBuilder.builder().withProjectDir(projectDir).build()
```

Comment:
This fixes a broken test which let's the project import fail if you have not set `ANDROID_HOME`.

It's acknowledged that the test set-up needs more work. Also, I do not see much value in the few current tests anyway. But this is beyond the scope of this PR imho. 

So either step up and provide a better fix or merge, we need the failing import fixed!